dist/tools/vera++/check.sh

```diff
@@ -13,8 +13,7 @@ CURDIR=$(cd "$(dirname "$0")" && pwd)
 
 . "$RIOTBASE"/dist/tools/ci/changed_files.sh
 FILEREGEX='\.([CcHh]|[ch]pp)$'
```

Comment:
This is the same as in `changed_files`. I don't think it is needed to set this either.

Reply:
removed